docs/lib/datastore/fragments/flutter/datastore-events.md

```diff
@@ -1,3 +1,17 @@
 ```dart
-TBD
+datastorePlugin.events.listenToDataStore((event) {
+    if (event["eventName"] == "networkStatus") {
+        setState(() {
+            networkIsUp = event["active"];
+        });
+    }
+});
```

Comment:
If this needs to be called before `addPlugin` or `configure` we should mention that here.

Reply:
Done.